### PR TITLE
ALT-aware mapping

### PR DESCRIPTION
**Pull request by marcelm**
The index files are now defined through a path with wildcards. They are no longer listed one by one, which makes it possible to have the GRCh37 version without and the GRCh38 with the .alt index file. This also simplifies the configuration.

With this change, BWA-MEM will run in ALT-aware mode.

This PR breaks the docker configuration files, but I will leave that to @MaxUlysse, who is working on #240 anyway.

### CODE REVIEW

File: configuration/uppmax-genomes.config [hunk 1]
```diff
@@ -23,13 +23,8 @@ params {
       snpeffDb      = "GRCh37.75"
       vardictHome   = "/sw/apps/bioinfo/VarDictJava/1.4.5/milou/VarDictJava/"
 
-      genomeAmb     = "${genome}.amb"
-      genomeAnn     = "${genome}.ann"
-      genomeBwt     = "${genome}.bwt"
+      bwaIndex      = "${genome}.*"  // this includes a few extra files not needed by BWA
```
Comment (MaxUlysse) [inline configuration/uppmax-genomes.config]: I corrected that with a `${genome}.{amb,ann,bwt,pac,sa}`

Comment (marcelm) [inline configuration/uppmax-genomes.config]: Does that work? Does `file()` support this type of wildcard? Also, we need the `.alt` file.

Comment (MaxUlysse) [inline configuration/uppmax-genomes.config]: Yes, we're using that already for some outputs: https://github.com/SciLifeLab/CAW/blob/master/main.nf#L140
Do we have alt for GRCh37 ?

Comment (marcelm) [inline configuration/uppmax-genomes.config]: No, that’s one reason why I used `.*`.

**Pull request merged**